### PR TITLE
mgmt, fluidrelay, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/fluidrelay/azure-resourcemanager-fluidrelay/src/test/java/com/azure/resourcemanager/fluidrelay/FluidRelayManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        try {
+            String serverName = "server" + randomPadding();
+            // @embedmeStart
+            server = fluidRelayManager.fluidRelayServers()
+                .define(serverName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withIdentity(new Identity().withType(ResourceIdentityType.NONE))
+                .withStoragesku(StorageSku.STANDARD)
+                .withProvisioningState(ProvisioningState.SUCCEEDED)
```
Comment (weidongxu-microsoft) [inline sdk/fluidrelay/azure-resourcemanager-fluidrelay/src/test/java/com/azure/resourcemanager/fluidrelay/FluidRelayManagerTests.java:85]: I don't think user need to set this.

Comment (v-hongli1) [reply sdk/fluidrelay/azure-resourcemanager-fluidrelay/src/test/java/com/azure/resourcemanager/fluidrelay/FluidRelayManagerTests.java]: Only the necessary basic settings have been retained in the new version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**